src/components/AccessibilityGuide/DynamicContent.stories.mdx

```diff
@@ -0,0 +1,204 @@
+import { Meta } from "@storybook/addon-docs";
+
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+
+<Meta title={getCategory("Dynamic Content")} />
+
+# Dynamic Content
+
+## Table of Contents
+
+- [General Information](#general-information)
+- [ARIA Live Regions](#aria-live-regions)
+- [Anti-Patterns](#anti-patterns)
+- [NYPL Patterns](#nypl-patterns)
+- [Resources](#resources)
+
+## General Information
+
+Most websites have elements on the page that are updated dynamically. For these
+scenarios, it should be clear to the user what the changes are and where the
+changes will take place. Typically, the update is indicated by displaying a visual
+loading animation before the new content is displayed. For most able-bodied users,
+this is usually not a problem but for users who use assistive technology, we
+need to make sure that the changes are understood and announced by the assistive
+technology.
+
+The quick takeaway for implementing this necessary functionality is to _always_
+render an empty container for the dynamic content in the DOM on the server-side.
+The container element should have an `aria-live="polite"` attribute applied. The
+following example shows one possible DOM structure that allows newly generated
+content to be recognized and announced by assistive technology.
+
+```html
+<div class="dynamic-content-container" aria-live="polite">
+  <!-- Any dynamic content should be inserted, removed, or updated here. -->
+</div>
+```
+
+## ARIA Live Regions
+
+The `aria-live` attribute is used to indicate that an element's content will be
+updated dynamically and that it should be announced to assistive technology.
+The `aria-live` attribute can be set to "off", "polite", or "assertive". _When_
+the content should be read by the assistant technology depends which of the three
```

Comment:
Nit: "assistive technology" vs "assistant technology"

Unless "assistant technology" is another name for it.